JS/Lesson35/onur/my-app/src/components/NewRecipe/index.jsx

```diff
@@ -67,144 +64,187 @@ export const NewRecipe = () => {
+								},
+								pattern: {
+									value: /\.(jpg|jpeg)$/,
+									message: 'Please enter a valid address'
+								}
+							})}
+							error={!!errors.strMealThumb}
+							helperText={errors.strMealThumb?.message}
+						/>
+
+						<TextField label='Tag Name' type="text"
+							{...register('strTags', {
+								required: {
+									value: true,
+									message: 'Tag name is required!'
+								},
+								minLength: 2,
+							})}
+							error={!!errors.strTags}
+							helperText={errors.strTags?.message}
+						/>
+
+						<TextField label='Youtube video address' type="text"
+							{...register('strYoutube', {
+								required: {
+									value: true,
+									message: 'Please add a YouTube video showing the preparation of your meal.'
+								},
+								pattern: {
+									value: /^(https?:\/\/)?(www\.)?(youtube\.com\/(?:[^/\n\s]+\/\S+\/|(?:v|e(?:mbed)?)\/|\S*?[?&]v=)|youtu\.be\/)([a-zA-Z0-9_-]{11})/,
+									message: 'Please enter a valid address'
+								}
+							})}
+							error={!!errors.strYoutube}
+							helperText={errors.strYoutube?.message}
+						/>
+
+						<TextField label='Ingredients' type="text"
+							{...register('ingredients', {
+								required: {
+									value: true,
+									message: 'Please enter the ingredients!'
+								},
+								minLength: 2,
+							})}
+							error={!!errors.ingredients}
+							helperText={errors.ingredients?.message}
+						/>
+
+						<TextField label='Measures' type="text"
+							{...register('measures', {
+								required: {
+									value: true,
+									message: 'Please enter the measures!'
+								},
+								minLength: 2,
+							})}
+							error={!!errors.measures}
+							helperText={errors.measures?.message}
+						/>
+
+						<TextField label='Source' type="text"
+							{...register('strSource', {
+								required: {
+									value: true,
+									message: 'Please enter your source!'
+								},
+								pattern: {
+									value: /^https:\/\//,
+									message: 'Please enter a valid address'
+								},
+								minLength: 2,
+							})}
+							error={!!errors.strSource}
+							helperText={errors.strSource?.message}
+						/>
+
+						<TextField label='Image source' type="text"
+							{...register('strImageSource', {
+								required: {
+									value: true,
+									message: 'Please enter the image source!'
+								},
+								pattern: {
+									value: combinedPattern,
+									message: 'Please enter a valid value'
+								},
+								minLength: 2,
+							})}
+							error={!!errors.strImageSource}
+							helperText={errors.strImageSource?.message}
+						/>
+
+						<FormControlLabel
+							label='Creative Commons'
+							control={
+								<Checkbox {...register('strCreativeCommonsConfirmed')} />}
+						/>
+
+						<Button sx={{ backgroundColor: 'primary.light', '&:hover': { backgroundColor: 'primary.main' } }} type='submit' variant="contained" color="success"  >ADD</Button>
```

Comment:
If you can avoid using plain strings in your code, do that. Since you can access colors from theme, use them form theme instead of string format: 'primary.light'.
To get access to theme here, you'd need to do as in this example: https://mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component